vignettes/irecist.Rmd

```diff
@@ -0,0 +1,858 @@
+---
+title: "Creating ADRS with iRECIST endpoints"
+output: 
+  rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Creating ADRS with iRECIST endpoints}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+
+library(admiral)
+
+link <- function(text, url) {
+  return(
+    paste0(
+      "[", text, "]",
+      "(", url, ")"
+    )
+  )
+}
+dyn_link <- function(text,
+                     base_url,
+                     relative_url = "",
+                     # Change to TRUE when admiral adopts multiversion docs
+                     is_multiversion = FALSE,
+                     multiversion_default_ref = "main") {
+  url <- paste(base_url, relative_url, sep = "/")
+  if (is_multiversion) {
+    url <- paste(
+      base_url,
+      Sys.getenv("BRANCH_NAME", multiversion_default_ref),
+      relative_url,
+      sep = "/"
+    )
+  }
+  return(link(text, url))
+}
+# Other variables
+admiral_homepage <- "https://pharmaverse.github.io/admiral/cran-release"
+
+library(admiraldev)
+```
+
+# Introduction
+
+This article describes creating an `ADRS` ADaM with oncology endpoint parameters
+based on iRECIST. It shows a similar way of deriving the endpoints presented in 
+[Creating ADRS (Including Non-standard Endpoints)](adrs.html). Most of the endpoints 
+are derived by calling `admiral::derive_extreme_event()`. Thus the examples in this 
+vignette can also be used as a starting point for implementing response criteria other 
+than iRECIST 1.1,  e.g., RECIST 1.1 or International Myeloma Working Group (IMWG) 
+criteria for the diagnosis of multiple myeloma.
+
+For more information on the iRECIST guidelines user may visit
+https://recist.eortc.org/recist-1-1-2/
+
+Examples are currently presented and tested using `ADSL` (ADaM) and
+`RS` (SDTM) inputs. However, other domains could be used. The functions 
+and workflow could similarly be used to create an intermediary `ADEVENT` ADaM.
+
+**Note**: *All examples assume CDISC SDTM and/or ADaM format as input
+unless otherwise specified.*
+
+# Programming Workflow
```

Comment:
In adrs.html (Creating ADRS (Including Non-standard Endpoints) there are parameters that are not included in the iRecist vignette. It may be worth mention that if we want to add them, they are described in Recist 1.1 vignette.

BCP	        Best Overall Response of CR/PR by Investigator (confirmation not required)
CBCP  	Best Confirmed Overall Response of CR/PR by Investigator
A1BOR	Best Overall Response by Investigator (confirmation not required) - Recist 1.1 adjusted for NED at Baseline
ACCB  	Alternative Confirmed Clinical Benefit by Investigator
OVRB	Overall Response by BICR
DEATH 	Death
LSTA  	Last Disease Assessment by Investigator
MDIS  	Measurable Disease at Baseline by Investigator